vertx-web/src/main/java/io/vertx/ext/web/Router.java

```diff
@@ -389,4 +389,14 @@ static Router router(Vertx vertx) {
    */
   @Fluent
   Router modifiedHandler(Handler<Router> handler);
+
+
+  /**
+   * Set whether the router should parse "forwarded"-type headers
+   *
+   * @param allow {@code false} to enable parsing of "forwarded"-type headers
```

Comment:
Shouldn't it be `true`?

Reply:
Not sure.Right now there is no security on header spoofing.

Reply:
Sorry, i mean the docs, shouldn't it be: `{@code true}` ;-)

Reply:
Sorry.Bad copy / paste. Removed it